source/adios2/toolkit/remote/remote_server.cpp

```diff
@@ -390,12 +403,7 @@ void PrepareResponseForGet(CMConnection conn, struct Remote_evpath_state *ev_sta
                 GetMsg->Dest,
                 GetMsg->GetResponseCondition,
                 acc,
-                var->m_Name,
```

Comment:
Changing this to pass in var, rather than the individual items makes the parameter list shorter, but I think it violates thread safety.  var is a data structure shared between threads.  Someone else may change the count, start, blockID, etc to support their own get, meaning that the info you feed into the operator in ReturnResponseThread might get trashed.  (It'd be lovely to see this show up in tsan, but the server isn't run for those tests.)

Reply:
Fantastic catch. This was the reason originally I had the long list of individual parameters.

Reply:
Experience has made me sensitive to our threading-related pain points, and info in vars is high on that list, so that sort of thing pops out...